OMCompiler/SimulationRuntime/c/simulation/simulation_info_json.c

```diff
@@ -110,7 +110,7 @@ static const char* skipValue(const char* str)
       switch (*str) {
       case '\0': fprintf(stderr, "Found end of file, expected end of string"); abort();
       case '\\':
-        if (str+1 == '\0') {
```

Comment:
This is what I found interesting, would the + operator to a pointer cause a implicit dereference in C?

Reply:
AFAIK, `ptr + 1` is merely a pointer to **next element** according to pointee type (that is, `&ptr[1]`).

Reply:
yea, I know. I found the original code a bit odd, that change makes it ptr[1] instead of &ptr[1] right? so there is a semantic change here. I do not know this area of the code base, so if someone else could elaborate here it would be good

Reply:
It is more clear if you parenthesize it, i.e, `ptr + 1` is the same  as `&(*(ptr+1))` which is the same as `&(ptr[1])`. 
`[]` has precedence over `&`, so you can say `&ptr[1]`. 

Just for fun: `ptr[1]` is the same as `1[ptr]`  :)

Reply:
Never the less, I think your change makes sense @atrosinenko